runtime/queries/rust/highlights.scm

```diff
@@ -156,16 +156,23 @@
   "macro_rules!"
 
   "let"
-  "ref"
-  "move"
 
-  "dyn"
-  "static"
-  "const"
   "async"
 ] @keyword
 
-(mutable_specifier) @keyword.mut
+(mutable_specifier) @keyword.storage.modifier.mut
+
+(reference_type "&" @keyword.storage.modifier.ref)
+(self_parameter "&" @keyword.storage.modifier.ref)
+
+[
+  (mutable_specifier)
```

Comment:
This is redundant since there's already a capture for this scope.

Reply:
Oops, you're right. I returned mutable as an additional scope but forgot to remove it from `storage.modifier`